core/field.js

```diff
@@ -906,3 +923,39 @@ Blockly.Field.prototype.getParentInput = function() {
 Blockly.Field.prototype.onBlocklyAction = function(_action) {
   return false;
 };
+
+/**
+ * Add the cursor svg to this fields svg group.
+ * @param {Element} cursorSvg The svg root of the cursor to be added to the
+ *     field group.
+ * @package
+ */
+Blockly.Field.prototype.setCursorSvg = function(cursorSvg) {
+  if (!cursorSvg) {
+    this.cursorSvg_ = null;
+    return;
+  }
+
+  if (this.fieldGroup_) {
+    this.fieldGroup_.appendChild(cursorSvg);
+    this.cursorSvg_ = cursorSvg;
+  }
+};
+
+/**
+ * Add the marker svg to this fields svg group.
+ * @param {Element} markerSvg The svg root of the marker to be added to the
+ *     field group.
+ * @package
+ */
+Blockly.Field.prototype.setMarkerSvg = function(markerSvg) {
+  if (!markerSvg) {
+    this.markerSvg_ = null;
+    return;
+  }
+
+  if (this.fieldGroup_) {
+    this.fieldGroup_.appendChild(markerSvg);
```

Comment:
This check here hints to me that this can be called before initialization. In which case, I don't see a code path in initialization that appends the ``this.markerSvg_`` to the field group during initialization.

Reply:
Hmm, yeah I don't think we actually need this or the check in the block_svg. Both should already be initialized by the time the user starts moving around.